entrypoint.spec.js

```diff
@@ -1,18 +1,19 @@
 import { expect } from 'chai'
 import isSvg from 'is-svg'
 import got from './core/got-test-client.js'
+import startServer from './server.js'
 
-let serverModule
+let server
 before(async function () {
   this.timeout('30s')
   // remove args coming from mocha
   // https://github.com/badges/shields/issues/3365
   process.argv = []
-  serverModule = await import('./server.js')
+  server = await startServer()
```

Comment:
The core problem I ran into here was that `cluster.fork()` attempts to spawn a new worker process that executes the same script as the primary process. When a worker process is spawned, it executes the same Node.js script that started the primary process, but this didn't have the desired effect under test because the primary process is the test runner.
I worked round this by
a) exporting a function we can test from `server.js`
b) only starting in multi-threaded mode if we are not under test
This means the actual multi-threaded server startup is **not** under test, but I couldn't find any way around this. I think it is a tradeoff we would have to accept if we go down this road.